framework/src/main/java/org/checkerframework/framework/flow/CFAbstractStore.java

```diff
@@ -71,6 +73,71 @@ public abstract class CFAbstractStore<V extends CFAbstractValue<V>, S extends CF
   /** Information collected about local variables (including method parameters). */
   protected final Map<LocalVariable, V> localVariableValues;
 
+  /** Stores after boolean variable assignment. */
+  protected final Map<LocalVariable, BooleanVarStore<V, S>> booleanVarStores;
+
+  /**
+   * An object that contains the then and else store after a boolean variable assignment.
```

Comment:
Here too, I think that "after a boolean variable assignment" is not the most important thing to document.
It would be helpful, here, to discuss the goal and the design.